sem_server_ros/URI subject
sem_server_ros/URI predicate
sem_server_ros/URI object
---
sem_server_ros/Triple feedback
